node/impl/full/eth.go

```diff
@@ -44,6 +44,11 @@ var ErrUnsupported = errors.New("unsupported method")
 
 const maxEthFeeHistoryRewardPercentiles = 100
 
+var (
+	// wait for 3 epochs
+	eventReadTimeout = 90 * time.Second
```

Comment:
there's a `EpochDurationSeconds` in chain/actors/builtin

Reply:
Fixed this.

Reply:
did you forget to push the fix? still has `90` hardwired here

```suggestion
	eventReadTimeout = builtin.EpochDurationSeconds * time.Second
```

Reply:
@rvagg I did but the tests barf on this because the tests have like 2ms block times and it's sometimes not enough for the head to get processed.

Reply:
hah, that's a bit of a dilemma then!